src/web/version_handler.go

```diff
@@ -0,0 +1,27 @@
+package web
+
+import (
+	"runtime"
+	"time"
+	"github.com/gin-gonic/gin"
+	"github.com/siesgstarena/epicentre/src/services/logger"
+)
+
+type version struct {
+    Name     		string `json:"name"`
+	Description     string `json:"description"`
+	Version 		string `json:"health"`
+	Timestamp       time.Time `json:"timestamp"`
+}
+
+// VersionHandler Sends info about version of API
+func VersionHandler(c *gin.Context)  {
+	logger.Log.Info("Inside Health Handler")
+	info := new(version)
+	info.Name = "epicentre"
+	info.Description = "Cloud Monitoring and Alerting Tool built by SIESGSTarena Platform Team)"
+	info.Version = runtime.Version()
+	loc, _ := time.LoadLocation("UTC")
+    info.Timestamp = time.Now().In(loc)
```

Comment:
`go fmt` here as well :)

Reply:
In my laptop their some issue of fmt I guess it doesn't always modify files.

But this indentation error is not seen in my laptop don't know-how